### PR TITLE
Allows Active Members to propose changes to the COC

### PR DESCRIPTION
**Pull request by SpencerCreveling**
Check one:
- [x] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Allows Active Members to propose changes to the COC

### CODE REVIEW

File: constitution.tex [hunk 1]
```diff
@@ -738,7 +738,7 @@
 \asubsubsubsection{Changes}
 The following process is used for making changes to the Code of Conduct document.
 \begin{enumerate}
-	\item A Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
+	\item An Active Member drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
```
Comment (mxmeinhold) [inline constitution.tex:741]:
```suggestion
	\item An Active Member or Root Type Person drafts a change to the Code of Conduct and makes publicly available both the summary and difference file of the change.
```

Comment (SpencerCreveling) [reply constitution.tex:741]: I see where you are coming from as a non-active RTP, however I feel that matters regarding the COC are primarily the concern of active members. Once an RTP leaves Active Member status they shouldn't concern themselves with the COC.

Comment (mxmeinhold) [reply constitution.tex:741]: This suggests that RTPs who were not active members have not been involved in day to day matters, which is simply not the case in recent memory.

Comment (geoffmoss0) [reply constitution.tex:741]: Reminder that not active != not at RIT

Comment (jabbate19) [reply constitution.tex:741]: IIRC Non-Active RTPs can't vote on Constitutional Amendments, and modifying the COC is of equal stature as the Constitution.

Comment (mxmeinhold) [reply constitution.tex:741]: Also, RTPs do not need to be active when they become RTPs

**Pull request closed without merging**